Dockerfile-dev

```diff
@@ -0,0 +1,17 @@
+########## Builder ##########
+FROM golang:1.21-alpine AS builder
+
+# Install the latest version of Delve
+RUN go install github.com/go-delve/delve/cmd/dlv@latest
```

Comment:
This seems like it could break if they change something upstream, sure you don't want to pin it?

Reply:
they're supposed to be backwards compatible but yeah that's a good idea can't trust it will be. I'll update quickly